### ISSUE TITLE
JAWS does not announce description as associated with aria-describedby for a fieldset with text inputs

### ISSUE THREAD

**Issue by alex-ju** [alphagov/reported-bugs#45]
**Upstream bug:** https://github.com/FreedomScientific/VFO-standards-support/issues/80
**Date:** 2018-05-09
**Reported by:** Alex Ju

---

### Overview

JAWS does not announce description as associated with `aria-describedby` for a fieldset that contains text inputs.

Example:
1. Go to https://codepen.io/alex-ju/full/KRQMvr/
2. Tab to an input inside the fieldset (i.e. the _Street address_ input)

### Expected result

We'd expect it to announce the description (basically to behave the same it does with a fieldset that contains radios or checkboxes - see example below)
1. Go to https://codepen.io/alex-ju/full/wjyrVe/
2. Tab to an input inside the fieldset (i.e. the _Yes_ input)

### Actual result

It doesn't announce the description.

### Example
https://codepen.io/alex-ju/full/wjyrVe/

### Additional Information

If `role="group"` is added to the fieldset, the description is being announced, but also the legend is being repeated which becomes too verbose.
https://codepen.io/alex-ju/full/PeQOOO/

We used this approach as [a workaround in our code](https://github.com/alphagov/govuk-frontend/pull/681).

#### JAWS version and build number
JAWS 18.0.4534 and 17.0.2729
 
#### Operating System and version
Windows 10 Pro

#### Browser and version:
Internet Explorer (IE) 11.371
